rapid_pro_tools/rapid_pro_client.py

```diff
@@ -143,9 +143,53 @@ def get_flow_definitions_for_flow_ids(self, flow_ids):
         :rtype: temba_client.v2.types.Export
         """
         return self.rapid_pro.get_definitions(flows=flow_ids, dependencies="all")
+    
+    def _get_archived_messages(self, created_after_inclusive=None,
+                                       created_before_exclusive=None):
+        """
+        Gets the raw messages from Rapid Pro's archives.
+        
+        Uses the created dates to determine which archives to download.
```

Comment:
Could you add a note explaining why we filter on created dates here (was briefly confused because we filter on modification date elsewhere). Something like

"Filtering is done on creation date because this is the only timestamp metadata field Rapid Pro supports filtering"